shard.yml

```diff
@@ -1,29 +1,29 @@
 name: lucky_flow
-version: 0.9.1
+version: 0.9.2
 
 authors:
   - Paul Smith <paulcsmith0218@gmail.com>
 
-crystal: ">= 1.4.0"
+crystal: "~> 1.4"
 
 license: MIT
 
 dependencies:
   selenium:
     github: crystal-loot/selenium.cr
-    version: ~> 0.12
+    version: ">= 0.12.1, < 0.13"
```

Comment:
Looks good, but just want to make sure that `0.12.1` is actually _required_ instead of `0.12.1`?

Reply:
Yeah, 0.12.0 has issues, so we need 0.12.1 at a minimum for this to work.